avdevice/__init__.py

```diff
@@ -469,136 +122,238 @@ def _wait(self, time_lapse):
             pass
 
     def _resetitem(self):
-        self._resetlock.acquire(timeout=2)
         try:
             resetting = None
             try:
-                response = self._sendingcommand.split(",")[2].split("|")  
-                self.logger.debug("Resetting {}: Searching for suiting command for sendingcommand response: {}.".format(self._name, response))  
-            except Exception as e:                    
-                response = self._send_commands[0].split(",")[2].split("|")                
-                self.logger.warning("Resetting {}: Cannot find Sendingcommand. Using first Command in queue: {}.".format(self._name, response))
-            for key in self._response_commands:                                
-                #self.logger.debug("Resetting {}: Trying to reset Item. Comparing: {} with {}".format(self._name, key,response))
+                response = self._sendingcommand.split(",")[2].split("|")
+                sendcommand = self._sendingcommand
+                if self._verboselevel >= 1:
+                    self.logger.debug("Resetting {}: Searching for suiting command for sendingcommand response: {}.".format(self._name, response))
+            except Exception as e:
+                response = self._send_commands[0].split(",")[2].split("|")
+                sendcommand = self._send_commands[0]
+                if self._verboselevel >= 2:
+                    self.logger.warning("Resetting {}: Cannot find Sendingcommand. Using first Command in queue: {}. Message: {}".format(self._name, response, e))
+            sorted_response_commands = sorted(self._response_commands, key=len, reverse=True)
+            for key in sorted_response_commands:
+                if self._verboselevel >= 2:
+                    self.logger.debug("Resetting {}: Trying to reset Item. Comparing: {} with {}".format(self._name, key, response))
```

Comment:
Does a check on `verboselevel > XY` makes sense? I would avoid such things and instead use the common logger configuration to configure debug logging.